README.md

```diff
@@ -5,143 +5,13 @@ There are currently five working endpoints:
-```
-<VirtualHost *:80>
-	#Change as appropriate
-        ServerName your-server-name.com 
-        Header set Access-Control-Allow-Origin "*"
-	#Change as appropriate
-        ServerAdmin webmaster@localhost
-
-        WSGIDaemonProcess dcc-dashboard-service threads=5
-        WSGIScriptAlias /api/v1 /var/www/html/dcc-dashboard-service/dcc-dashboard-service.wsgi
-        <Directory dcc-dashboard-service>
-                 WSGIProcessGroup dcc-dashboard-service
-                 WSGIApplicationGroup %{GLOBAL}
-                 Order deny,allow
-                 Allow from all
-        </Directory>
-        ErrorLog ${APACHE_LOG_DIR}/error.log
-        CustomLog ${APACHE_LOG_DIR}/access.log combined
-</VirtualHost>
-```
-
-This will set the web service endpoint under ‘your-server-name.com/api/v1’ 
-
-Save it and run 
-```
-sudo service apache2 restart
-```
-## Setting Up HTTPS Using 'LetsEncrypt'
-If you want to set up HTTPS for the web service, you can do so by using LetsEncrypt. 
-Before starting however, you must comment out the `WSGIDaemonProcess` line. Not doing so will cause an error when trying to install the SSL certificates via Certbot.  
-Go to https://certbot.eff.org/ and for software, choose Apache. For the system, choose as appropriate. Follow their installation instructions. 
-Once the whole installation process has concluded, go back to `/etc/apache2/sites-enabled/000-default.conf` and uncomment the `WSGIDaemonProcess` line, so it is active once again. The daemon will serve both port 80 and 443 (HTTP and HTTPS respectively). 
-Finally, run:
-```
-sudo service apache2 restart
-```
-## Installing Pip and Flask (If you don't have them)
-You want to have pip installed so you can install various python packages:
-```
-#Install pip
-sudo apt-get install python-pip
-```
-Then, to install Flask, do:
-```
-#Install flask
-sudo pip install flask
-```
-## Setting up the virtualenv for the Flask App
-The Web Service Flask app requires various python packages. To avoid any conflicts with other web applications within the same machine, we will create a virtual environment with the required packages to run. Assuming you are in the root directory of your machine, you can set up the virtual environment as follows:
-```
-#Make the virtual environment and install requirements
-cd dcc-dashboard-service
-virtualenv env
-. env/bin/activate 
-pip install -r requirements
-```
-## Enabling the 'instructions.sh' Script
-To enable the instructions.sh script, which contains the commands to run and update the Elasticsearch indicies that the web service uses, do:
-```
-chmod +x dcc-dashboard-service/instructions.sh
-```
-## Setting the 'instructions.sh' as a cron job
-To start the cron job, simply do:
-```
-crontab dcc-dashboard-service/crontab.txt
-```
-<b>NOTE:</b> crontab.txt assumes you have an environmental variable called `$REDWOOD_ACCESS_TOKEN`. To set up that variable, edit your bash file (in my case it was .bashrc). At the end of it, add
```

Comment:
add a script that checks for the existence of $REDWOOD_ACCESS_TOKEN? found some ideas here: https://stackoverflow.com/questions/39296472/shell-script-how-to-check-if-an-environment-variable-exists-and-get-its-value